### PR TITLE
Update send message

### PR DESCRIPTION
**Pull request by Eliada**
**Change Log** :memo:

 * Update send message API: WhatsApp version 2.18.16
 * text message:

 before:
```json
{
  "payload": {
    "to":   "username",
    "body": "body"
  }
}
```
now:
```json
{
  "recipient_type": "individual",
  "to": "wa_id",
  "type": "text",
  "text": {
    "body": "body"
  }
}
```
   * hsm message:

```json
{
  "payload": {
    "to": "username",
    "hsm": {
      "namespace": "namespace",
      "element_name": "element_name",
      "localizable_params": "params"
    }
  }
}
```
now:
```json
{
  "type": "hsm",
  "recipient_type": "individual",
  "to": "wa_id",
  "hsm": {
    "namespace":          "namespace",
    "element_name":       "element_name",
    "fallback_lg":        "pt",
    "fallback_lc":        "BR",
    "localizable_params": "params"
  }
}```

### CODE REVIEW

File: lib/whats/actions/send_hsm_message.rb [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
   module Actions
     class SendHsmMessage
-      PATH = "/api/rest_send.php"
+      PATH = "/v1/messages"
 
-      def initialize(client, username, namespace, element_name, params)
+      def initialize(client, wa_id, namespace, element_name, params)
         @client       = client
-        @username     = username
+        @wa_id        = wa_id
```
Comment (restarac) [inline lib/whats/actions/send_hsm_message.rb]: `wa_id = whatsapp_id` foi so para ficar mais curto?

Comment (Eliada) [reply lib/whats/actions/send_hsm_message.rb]: `wa_id` is the name of the key used in the http response from WhatsApp, just keeping the same name in our implementation

File: spec/whats/actions/send_hsm_message_spec.rb [hunk 2]
```diff
@@ -21,29 +21,25 @@
...
+    #   before { stub_send_hsm_message_with_unknown_contact_response(wa_id, namespace, element_name, params: params) }
+    #
+    #   it "returns payload as nil" do
+    #     expect(action.call["payload"]).to be_nil
+    #   end
+    #
+    #   it "returns error unknown contact" do
+    #     expect(action.call["error"]).to eq "errorcode" => 404, "errortext" => "unknown contact"
+    #   end
+    # end
```
Comment (dynaum) [inline spec/whats/actions/send_hsm_message_spec.rb]: 😱

**Comment by dynaum** [comment]
Pq é o @Eliada que abre o PR mas é o @Eliada2 que comita?

File: spec/whats/actions/send_hsm_message_spec.rb (hunk 2 above)
Comment (Eliada) [reply spec/whats/actions/send_hsm_message_spec.rb]: As the commit message says:
>WIP Still don't know the new erros response format 

This is a reminder that this test has to be working in the near future, but more information is needed to know the error output

Comment (Eliada) [reply spec/whats/actions/send_hsm_message_spec.rb]: But you are right there's pending tests for that :sweat_smile:

**Comment by Eliada** [comment]
> Pq é o @Eliada que abre o PR mas é o @Eliada2 que comita?
Is the difference between  user email on git config. Multiple emails multiple github commiters

**Pull request merged**